metricbeat/module/mysql/query/_meta/fields.yml

```diff
@@ -0,0 +1,9 @@
+- name: query
+  type: group
+  description: >
+    query
+  fields:
+    - name: example
+      type: keyword
```

Comment:
This is not quite right, I think, because the resulting object is `mysql.test.example` not `mysql.query.example`, right?